content/showcase/_index.html

```diff
@@ -12,6 +12,14 @@ <h1 class="title is-2">UQCS Showcase</h1>
         <div class="columns is-centered">
             <div class="column is-8">
                 <div class="showcase-gallery columns is-multiline">
+                    {{< showcase-item
+                        src="/img/media/showcase/nie.png"
```

Comment:
One last thing (sorry!) it looks like your image is a jpg but this references a png. Either should be fine, it just needs to be consistent.

Reply:
```suggestion
                        src="/img/media/showcase/nie.jng"
```

Reply:
Changed the extension.